dags/multipod/maxtext_nightly.py

```diff
@@ -34,14 +35,20 @@
 ) as dag:
   default_test_name = "maxtext-nightly"
 
+  test_mode = SetupMode.NIGHTLY
+  test_platform = Platform.GCE
+
   # Maxtext
   maxtext_nightly_1slice_v4_8 = maxtext_gce_config.get_maxtext_nightly_config(
       tpu_version=TpuVersion.V4,
       tpu_cores=8,
       tpu_zone=Zone.US_CENTRAL2_B.value,
       time_out_in_min=60,
       is_tpu_reserved=False,
+      automated_test=True,
```

Comment:
`True` is the default, so no need to override here.

Also, I wonder if we can just infer `automated_test` from `composer.is_prod_env()` instead of manually configuring it - will we ever be triggering manual tests in the prod environment? https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/7bd980b7cbf966ed5b4bfd902c9785eb0a968108/dags/composer_env.py#L31

Reply:
Done. 

I might still want to keep `automated_test` because I might want to simulate what would happen in prod when manually trigger the test in the future.